diff -ruN libwebp-0.6.0.orig/Android.mk libwebp-0.6.0/Android.mk
--- libwebp-0.6.0.orig/Android.mk	2017-10-22 10:55:55.645170270 -0700
+++ libwebp-0.6.0/Android.mk	2017-10-22 12:31:09.944186793 -0700
@@ -259,7 +259,7 @@
 
 WEBP_SRC_PATH := $(LOCAL_PATH)
 include $(WEBP_SRC_PATH)/imageio/Android.mk
-include $(WEBP_SRC_PATH)/examples/Android.mk
+#include $(WEBP_SRC_PATH)/examples/Android.mk
 
 ifeq ($(USE_CPUFEATURES),yes)
   $(call import-module,android/cpufeatures)
